src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -6,7 +6,222 @@ stub: true
+
+If you pick one of the available templates (such as 'blog'), additional collections will be generated for you. You can then skip the following section, because you already have a collection other than "Users" that you can use.
+
+If you don't pick any templates, only a single collection called "Users" will be generated.
+
+If you start the PayloadCMS server using `npm run dev`, you can go to `http://localhost:3000/admin` to enter your PayloadCMS Dashboard for the first time. Create the initial administrator account and enter the Dashboard. You should now see that the "Users" collection contains just the administrator account.
+
+#### Creating a PayloadCMS Collection
+
+You can create a new Collection called `Posts.ts` in `src/collections`.
+
+```astro title="src/collections"
+import { CollectionConfig } from "payload/types";
+
+const Posts: CollectionConfig = {
+  slug: "posts",
+  admin: {
+    useAsTitle: "title",
+  },
+  access: {
+    read: () => true,
+  },
+
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "content",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "slug",
+      type: "text",
+      required: true,
+    },
+  ],
+};
+
+export default Posts;
+```
+
+This creates a "Posts" collection which requires the following : `title`, `content`, `slug`. You will use this collection to manage your blog posts. The colllection is publicly available, so you shouldn't put any sensitive info inside.
+
+You then need to import and add it to the available collections in the `payload.config.ts` file.
+
+```astro title="src/payload.config.ts"
+import { buildConfig } from "payload/config";
+import path from "path";
+
+import Users from "./collections/Users";
+import Posts from "./collections/Posts";
+
+export default buildConfig({
+  serverURL: "http://localhost:3000",
+  admin: {
+    user: Users.slug,
+  },
+  collections: [Users, Posts],
+  typescript: {
+    outputFile: path.resolve(__dirname, "payload-types.ts"),
+  },
+  graphQL: {
+    schemaOutputFile: path.resolve(__dirname, "generated-schema.graphql"),
+  },
+});
+```
+
+If you then go into the PayloadCMS Dashboard, you should see a new collection called "Posts" appear next to the "Users" collection. Enter the "Posts" collection and create a new post. After saving it, you will notice the API URL appear in the bottom right corner.
+
+If not already running, start the server using `npm run dev` again.
+
+Open `http://localhost:3000/api/posts` in your browser. You should see a JSON file containing the post you have created as an object.
+
+:::tip
+By default, both Astro and PayloadCMS will use the port 3000. You might want to change the PayloadCMS port in the `src/server.ts` file. Don't forget to update the `serverURL` in `src/payload.config.ts` as well.
+:::
+
+```astro title="http://localhost:3000/api/posts"
```

Comment:
Note to Team Docs: 

Have we ever used a *website URL* as a code snippet title before?  I'm not sure what our precedent here is, when you want to tell the reader to navigate to a URL but then what they should see is a code snippet!  :smile:

What do we think? Do we need anything here?